models/pull.js

```diff
@@ -28,6 +28,18 @@ function Pull(data, signatures, comments, commitStatus, labels) {
       this.data.closes = data.closes;
       this.data.connects = data.connects;
    }
+
+   this.isPassing = function() {
```

Comment:
If this is only really meant to be called during the constructor, what do you think about making it private? If I saw that the Pull object had both a `passing` field and an `isPassing` method I don't think it'd be immediately obvious as to which one is the canonical representation of whether or not a pull is passing.

You could keep the name, but just define it within the scope of the `pull` module without placing it on the actual Pull object.

Reply:
Yes, it only needs to be calculated when build statuses change, which they never do other than on construction.

I'm game for making it private, I agree it would make it much more clear which one should be used.